Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/TrisoftCmdlet.cs

```diff
@@ -35,7 +35,7 @@ namespace Trisoft.ISHRemote.Cmdlets
     ///                and: http://technet.microsoft.com/en-us/library/dd819489.aspx
     /// Progress Record: http://community.bartdesmet.net/blogs/bart/archive/2006/11/26/PowerShell-_2D00_-A-cmdlet-that-reports-progress-_2D00_-A-simple-file-downloader-cmdlet.aspx
     /// </summary>
-    public abstract class TrisoftCmdlet : Cmdlet
+    public abstract class TrisoftCmdlet : PSCmdlet
```

Comment:
I rather stay with Cmdlet. Switching to PSCmdlet means PowerShell runtime dependencies come in. So the idea of using the DLL as library is affected as you need a PowerShell runtime. The PSCmdlet to Cmdlet switch was done on 20160202 as part of TS-10630, so this would be a rollback (can't find argumentation why it was done back then besides the above).
Probably means switching the above ParameterSet.Name to an actual property check on IssuerUrl and MexUrl. Not a good enough reason to introduce PSCmdlet over Cmdlet as base class.